client/pages/cart.jsx

```diff
@@ -46,9 +51,30 @@ export default class Cart extends React.Component {
     })
       .then(res => res.json())
       .then(data => {
-        this.props.resetCart();
+        const orderId = data.orderId;
+        const emailElement = React.createElement(Email, { orderId, items });
+        const content = ReactDOMServer.renderToStaticMarkup(emailElement);
+
+        axios({
+          method: 'POST',
+          url: 'http://localhost:3001/send',
```

Comment:
Avoid hard-coding your host/port. `/api/send`  should work. You may need to update your route.